account_payment_term_security/security/security.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="account_payment_term_mgmt" model="res.groups">
+        <field name="name">Enable to manage payment terms</field>
+        <field name="category_id" ref="base.module_category_accounting_accounting" />
```

Comment:
This can't be in accounting, as it creates that this is no more a dropdown. It should be on the technical category.